src/components/header/NavBar.vue

```diff
@@ -57,16 +57,13 @@ const hasNext = computed<boolean>(() => {
   }
   return true;
 });
-const labels = computed<Labels>(() => configStore.config.labels || {
-  manifest: 'manifest',
-  item: 'item',
-});
+
 const nextButtonLabel = computed<string>(() => (itemIndex.value === manifest.value.sequence.length - 1
-  ? `${t('next')} ${t(labels.value.manifest ? labels.value.manifest : 'Manuscript')}`
-  : `${t('next')} ${t(labels.value.item)}`));
+  ? `${t('next')} ${t('Manuscript')}`
+  : `${t('next')} ${t('item')}`));
 const prevButtonLabel = computed<string>(() => (itemIndex.value === 0
-  ? `${t('prev')} ${t(labels.value.manifest ? labels.value.manifest : 'Manuscript')}`
-  : `${t('prev')} ${t(labels.value.item)}`));
+  ? `${t('prev')} ${t('Manuscript')}`
```

Comment:
Ideally, this looks like this `t('previous_manifest')`. No combined translations anymore. We talked about this, you remember?

Reply:
yes you are right, we talked about this, but it seems that I had a misunderstanding, but now it's clear.